Sasquatch/Sasquatch/ViewControllers/Sections/SimplePropertiesTableSection.swift

```diff
@@ -15,8 +15,10 @@ class SimplePropertiesTableSection : PropertiesTableSection {
 
     // Set cell to respond to being edited.
     cell.keyField.addTarget(self, action: #selector(propertyKeyChanged), for: .editingDidEnd)
+    cell.keyField.addTarget(self, action: #selector(recordCurrentTarget), for: .editingDidBegin)
+    
```

Comment:
Can remove this line, we usually have empty lines within code blocks only before a comment.

Reply:
removed